# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/reiserfsprogs/hotfix-ll-limits.patch.musl
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- reiserfsprogs-3.6.27/resize_reiserfs/resize_reiserfs.c.vanilla	2025-04-09 11:17:17.574485463 +0200
+++ reiserfsprogs-3.6.27/resize_reiserfs/resize_reiserfs.c	2025-04-09 11:17:30.673574605 +0200
@@ -41,7 +41,7 @@
 
 	/* Some error occured while convertion or the specified
 	   string is not valid. */
-	if (bytes == LONG_LONG_MIN || bytes == LONG_LONG_MAX ||
+	if (bytes == LLONG_MIN || bytes == LLONG_MAX ||
 	    (bytes_str != end && bytes_str != end + 1))
 		return -EINVAL;
 
